src/style.scss

```diff
@@ -112,3 +112,7 @@ wc-ribbon-cell.ribbon__subject--cell--no-annotation {
   }
 }
 
+.table__row__supercell__cell__link {
+  color: rgb(38, 129, 166)
```

Comment:
Could you use the `$link-color` theme variable instead so that we don't have hardcoded colors scattered about?

Reply:
Done